assets/scripts/store/reducers/errors.js

```diff
@@ -17,9 +17,7 @@ const status = (state = initialState, action) => {
         abortEverything: action.abortEverything
       }
     case HIDE_ERROR:
-      return {
-        ...initialState
-      }
+      return initialState
```

Comment:
Should we be worried that referencing `initialState` instead of cloning it could cause `initialState` to be accidentally mutable by other code?

Reply:
I don't believe this puts initialState more at risk of accidental changes. Without using a lib such as ImmutableJS to force immutability, the `initialState`, as well as the general `state` variables are accidentally mutable.

I saw that a couple of other reducers follow this pattern as well:
[dialogs.js](https://github.com/streetmix/streetmix/blob/master/assets/scripts/store/reducers/dialogs.js#L16)
[menus.js](https://github.com/streetmix/streetmix/blob/master/assets/scripts/store/reducers/menus.js#L16)